src/core/config/factorization/inline-plugin.js

```diff
@@ -5,7 +5,6 @@
 const InlinePluginFactorization = (options) => () => ({
   fn: options.fn,
   components: options.components,
-  state: options.state,
```

Comment:
This does nothing. It get's assigned to system instance as arbitrary data but it's never used. IMHO this was replaced later by `initialState` config option and was forgotten.